samples/jira-pr.js

```diff
@@ -0,0 +1,175 @@
+/*
+* Creates a pull request on GitHub when a Jira issue is opened.
+*
+* Setup a Jira Webhook to your ngrok/fission installation:
+* E.g. 
+*   URL http://9560d9c6.ngrok.io/gh/applets/jira-pr
+*   Events
+*   Issue related events
+*   JQL: PROJECT="Your Project Name"
+*   Issue: created, updated
+* 
+* Configure the constants below based on your needs.
+*/
+
+const octokit = require("@octokit/rest")({
+  // custom GitHub Enterprise URL
+  baseUrl: "http://github.gitaboard.com/api/v3"
+});
+const jsonwebtoken = require("jsonwebtoken");
+const fs = require("fs");
+
+// Constants
+const JIRA_ISSUE_LABEL = "develop"; // Matches a specific Jira label
+const REPO_NAME = "jira-pr-tests"; // The name of your repository
+const REPO_OWNER = "chewbacca"; // The repository owner (user or org)
+const BASE_BRANCH = "master"; // target branch for PRs
+
+// Secrets
+const appId = parseInt(
```

Comment:
From my previous comment, something like:

`const appID = process.env.GH_APP_ID` or some derivative thereof.

Does fission bootstrap env vars for you in the UI, or on the CLI?

Reply:
Secrets are mounted as a volume which we are loading and parsing as if it was a file. It’s the recommended approach for fission functions!

Reply:
Also to answer your question I think it's possible to expose secrets and configmaps in K8S as ENV variables but I am not sure that's the supported way for Fission functions.